test/raygun_batch_lifecycle_test.js

```diff
@@ -12,7 +12,7 @@ test("batch transport doesn't keep processes alive", async (t) => {
           return reject(error);
         }
 
-        t.assert(process.exitCode === 0);
```

Comment:
`assert` is no longer part of `tap`, replace by methods like `ok()` or `equal()`